### DISCUSSION TITLE
[ Implemented ]  Ability to define parameters via Spreadsheet/User Parameters

### DISCUSSION THREAD

**Discussion by erikringwalters** [hlorus/CAD_Sketcher#238] [Feature Requests]
A common CAD workflow relies heavily on defining the parameters of the user's model as they build it.

A feature similar to FreeCAD's Spreadsheets or F360's User Parameters in CAD Sketcher would be incredibly useful.

**Comment by amrsoll** [comment]
There is now support for "drivers" which allow you to have user defined values / parametric modeling :) see 0.25 release and the makertales video

**Comment by TheMakerTales** [comment]
This feature is now implemented through the support of blender drivers.
